### PR TITLE
Update quart-trio and hypercorn links

### PR DESCRIPTION
**Pull request by decorator-factory**
It seems like the projects were moved from gitlab to github.

### CODE REVIEW

**Review by A5rocks — APPROVED** [review]
Yeah this is good, though someone should tell @pgjones their site has incorrect links to quart and hypercorn! (Not quart-trio though)

**Comment by decorator-factory** [comment]
Huh, getting a typo-fix merged is enough to get merge rights? That is terrifying :frowning_face:

**Comment by CoolCat467** [comment]
Well, you have to start somewhere. Reviews must be made an a pull request before it is merged, CI must succeed to be able to merge, the main branch is protected from force pushes, and even if an issue were merged into main we can always revert it.

**Pull request merged**
